appveyor.yml

```diff
@@ -24,7 +24,7 @@ test_script:
   - npm test
 
 after_test:
-  - ps: If ($env:nodejs_version -eq "Current") { npm run prebuild }
+  - ps: If ($env:nodejs_version -eq "8") { npm run prebuild }
```

Comment:
This seems to be run unconditionally. Not sure if that's ok or if we should add e.g. `version-unchanged` to `package.json`.

Reply:
Might be good to keep this, to know that prebuilds are working (and not find out they're broken on release)? idk

Reply:
Aye, lets just get this ball rolling first and tweak as we go. Still kind of figuring this out.

Reply:
If we don't want this behavior, then an `on: ..` could work (same as under `deploy:`)